BrushedMotorControllerFirmware/include/uart_shell/shell_direct_motor_control.c

```diff
@@ -140,17 +146,45 @@ static int cmd_actual_direction(const struct shell *shell, size_t argc, char *ar
 }
 
 SHELL_CMD_ARG_REGISTER(speed, NULL,
-	"speed in milli RPM (one thousands of RPM)\nspeed to get speed\n speed <val> to set speed",
-	cmd_speed, 1, 1);
+#if defined(CONFIG_SPEED_CONTROL_ENABLE)
+			"Speed in milli RPM (one thousands of RPM).\n"
+			"speed to get current motor speed.\n speed <val> to set new target speed.",
```

Comment:
```suggestion
			"speed to get current motor speed.\n"
			"speed <val> to set new target speed.",
```

to make it more readable from src code